src/Core/Components/Dialog/FluentDialogHeader.razor.cs

```diff
@@ -11,6 +8,9 @@ public partial class FluentDialogHeader : FluentComponentBase
     [CascadingParameter]
     private FluentDialog Dialog { get; set; } = default!;
 
+    /// <summary />
+    private int TabIndex { get; set; } = 0;
```

Comment:
You can define this property as a nullable Parameter property. So, to avoid adding another `ExcludeFromTabFocus` property and to allow more customization for the developer.
```csharp
[Parameter]
public int? TabIndex { get; set; } = 0;
```